Authenticators/LDAP/LDAPauth.ini

```diff
@@ -35,6 +35,9 @@ group_attr = uniqueMember
 ; Uncomment to provide list of registered users from LDAP
 ; provide_users = true
 
+; Uncomment to use StartTLS without cert check
```

Comment:
I think we should be more explicit here to make absolutely sure the limitations are understood. Maybe something like:
```ini
; Uncomment to use StartTLS
; WARNING:
;  While this option will enforce an encrypted connection it does
;  not check the server certificate. Man-in-the-middle is possible.
```